### ISSUE TITLE
Include slot name in WebJobRunIdentifier

### ISSUE THREAD

**Issue by phatcher** [Azure/azure-webjobs-sdk#1332]
One of our release scripts was mis-configured with an incorrect environment value for AzureWebJobsDashboard/AzureWebJobsStorage; This caused two environments to break, since we had two environments competing for the same storage locks.

Took a little while to determine what was going on as there's a fair few project/environments but eventually tracked it down using the information in the azure-jobs-host-output file.

This includes a WebJobRunIdentifier structure as follows...

```
  "WebJobRunIdentifier": {
    "WebSiteName": "MySite",
    "JobType": "Continuous",
    "JobName": "MyWebJob",
    "RunId": ""
  }
```
If this could be extended with the slot name, it would make finding this sort of mis-configuration much faster

**Comment by paulbatum** [comment]
Makes sense. This is not going to be a high priority for us but if you sent at PR we'd consider it.
